### DISCUSSION TITLE
What's your "Take a picture, upload it to Immich, archive off of you phone" workflow?

### DISCUSSION THREAD

**Discussion by Stefomat** [immich-app/immich#8369] [Q&A]
Hi all,

I love Immich and have been using it since I found it.
Over time, I've accumulated quite a few pictures and videos on my phone (5000+), all uploaded to Immich via the app. Now that my storage is running low, I'm wondering "What to do with the old photos?".

Before Immich, I used to move the oldest photos from my phone to my NAS at irregular intervals, in folders like 2020, 2021, 2022, etc.

I could continue like this and move old photos from my phone. Immich would then show them as "remote only".

BUT, I actually want to include older photos that are already in my archive and have never been seen by Immich as an external library too. Would this get in each other's way? Would Immich then display photos multiple times if I move them from my smartphone to the archive folder?

What is your way of tidying up old photos but still keeping them in Immich?

Thanks

**Comment by jacobmee** [comment]
Surely you can use external lib, which works fine.  The problem is upload library.  It'll be mess after moving your file out, can't find photo or thumbs, you'll need to do a full scan (modify scan doesn't work for me) after each big move, which is not ideal, but surely can work.
